### PR TITLE
Update vendor scripts and Rails dependency

### PR DESCRIPTION
**Pull request by malandrina**
- Pull latest `jquery.payment.js` from `stripe/jquery.payment#master`
- Update Rails dependency to 4.2.2

### CODE REVIEW

File: jquery-payment-rails.gemspec [hunk 1]
```diff
@@ -16,5 +16,5 @@ Gem::Specification.new do |gem|
   gem.executables   = gem.files.grep(%r{^bin/}).map{ |f| File.basename(f) }
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ["lib"]
-  gem.add_dependency "railties", "~> 4.0.0"
+  gem.add_dependency "railties", "~> 4.2.0"
```
Comment (gabebw) [inline jquery-payment-rails.gemspec]: The commit message says:

> Update Rails dependency to 4.2.2

Should this line be 4.2.2?

Comment (malandrina) [inline jquery-payment-rails.gemspec]: Yes! Done: https://github.com/thoughtbot/jquery-payment-rails/commit/98969dc20b48f2288105400e36e37c2989f57d74

**Comment by gabebw** [comment]
:+1:

**Comment by keoghpe** [comment]
Some one should really merge this pull request

**Comment by hatboyzero** [comment]
Someone should really tag a new version so that these updates are available via 'bundle install' without having to point to a specific changeset in the git repo in the Gemfile.

**Pull request merged**